pkg/infra/teams/team1/social_capital.go

```diff
@@ -6,20 +6,6 @@ import (
 	"math"
 )
 
-// Function which defines how an agent perceives an action
-func actionSentiment(action decision.FightAction) [4]float64 {
-	switch action {
-	case decision.Cower:
-		return [4]float64{0.0, 0.0, -0.1, -0.1}
-	case decision.Attack:
-		return [4]float64{0.0, 0.0, 0.1, 0.1}
-	case decision.Defend:
-		return [4]float64{0.0, 0.0, 0.1, 0.1}
-	default:
-		return [4]float64{0.0, 0.0, 0.0, 0.0}
-	}
-}
-
 // Ensures a float is between -1 and 1
 func boundFloat(inputNumber float64) float64 {
```

Comment:
I think a lot of functions in this file can be moved to `helper_functions`

Reply:
Agree, will do that now.